_api-reference/cat/cat-nodes.md

```diff
@@ -31,7 +31,7 @@ GET _cat/nodes
 
 All cat nodes URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameters:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameters:
```

Comment:
`[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)`